lib/modules/datasource/types.ts

```diff
@@ -3,6 +3,7 @@ import type {
   CustomDatasourceConfig,
 } from '../../config/types';
 import type { ModuleApi } from '../../types';
+import type { CandidateReleaseConfig } from '../../workers/repository/process/lookup/types';
```

Comment:
I don't like the idea to reference types from worker here 🤔 it's a kind of circular type reference

Reply:
Yes, actually I'll move most of the code to datasource